### DISCUSSION TITLE
x-reactions 中的变量

### DISCUSSION THREAD

**Discussion by Zwahxf** [alibaba/formily#3009] [Q&A 2.x] [answered]
"{{$self.value === '123'}}" 123如果是个变量怎么写呢

**Comment by janryWang** [comment, marked as answer]
`<SchemaField schema={{ ... "{{$self.value === aaa}}" ... }} scope={{aaa:123}}>...</SchemaField>`

**Reply by kongjunAI** [reply]
这样情况下，如何在编辑器中可视化配置呢？

**Reply by kongjunAI** [reply]
![image](https://user-images.githubusercontent.com/76086371/229424898-6eb129c0-c22a-4402-9d8a-da994853b927.png)
